core/src/display_object/movie_clip.rs

```diff
@@ -3473,6 +3477,25 @@ impl<'gc, 'a> MovieClipData<'gc> {
         Ok(())
     }
 
+    #[inline]
+    fn define_scaling_grid(
```

Comment:
Should we also add a stub here, like we do in the actionscript setter?

Reply:
Oh, this was a bit late. I don't know, maybe? Any second opinions...?

Reply:
I'd say not, this is doing what it's supposed to